2021-10-01 09:58:52,769 INFO    MainThread:5358 [wandb_setup.py:_flush():69] setting env: {}
2021-10-01 09:58:52,769 WARNING MainThread:5358 [wandb_setup.py:_flush():69] could not find program at scripts/train.py
2021-10-01 09:58:52,769 INFO    MainThread:5358 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-01 09:58:52,769 INFO    MainThread:5358 [wandb_init.py:_log_setup():348] Logging user logs to /home/bhuwan/Desktop/cat-dog/outputs/2021-10-01/09-58-52/wandb/run-20211001_095852-1hyfj0ab/logs/debug.log
2021-10-01 09:58:52,769 INFO    MainThread:5358 [wandb_init.py:_log_setup():349] Logging internal logs to /home/bhuwan/Desktop/cat-dog/outputs/2021-10-01/09-58-52/wandb/run-20211001_095852-1hyfj0ab/logs/debug-internal.log
2021-10-01 09:58:52,769 INFO    MainThread:5358 [wandb_init.py:init():381] calling init triggers
2021-10-01 09:58:52,769 INFO    MainThread:5358 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {'lr': 0.0003, 'base_filters': 32, 'epochs': 2}
2021-10-01 09:58:52,769 INFO    MainThread:5358 [wandb_init.py:init():430] starting backend
2021-10-01 09:58:52,769 INFO    MainThread:5358 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-01 09:58:52,774 INFO    MainThread:5358 [backend.py:ensure_launched():135] starting backend process...
2021-10-01 09:58:52,777 INFO    MainThread:5358 [backend.py:ensure_launched():139] started backend process with pid: 5390
2021-10-01 09:58:52,777 INFO    MainThread:5358 [wandb_init.py:init():435] backend started and connected
2021-10-01 09:58:52,778 INFO    MainThread:5358 [wandb_init.py:init():494] updated telemetry
2021-10-01 09:58:52,779 INFO    MainThread:5358 [wandb_init.py:init():517] communicating current version
2021-10-01 09:58:53,297 INFO    MainThread:5358 [wandb_init.py:init():522] got version response upgrade_message: "wandb version 0.12.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-01 09:58:53,297 INFO    MainThread:5358 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-01 09:58:53,583 INFO    MainThread:5358 [wandb_init.py:init():557] starting run threads in backend
2021-10-01 09:58:57,681 INFO    MainThread:5358 [wandb_run.py:_console_start():1605] atexit reg
2021-10-01 09:58:57,681 INFO    MainThread:5358 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-10-01 09:58:57,681 INFO    MainThread:5358 [wandb_run.py:_redirect():1484] Redirecting console.
2021-10-01 09:58:57,682 INFO    MainThread:5358 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-01 09:58:57,682 INFO    MainThread:5358 [wandb_init.py:init():582] run started, returning control to user process
2021-10-01 09:59:02,594 INFO    MainThread:5358 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-10-01 09:59:02,594 INFO    MainThread:5358 [wandb_run.py:_restore():1547] restore
2021-10-01 09:59:05,239 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 654
  total_bytes: 654
}

2021-10-01 09:59:05,657 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 654
  total_bytes: 654
}

2021-10-01 09:59:06,599 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 654
  total_bytes: 681
}

2021-10-01 09:59:06,701 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 654
  total_bytes: 28065
}

2021-10-01 09:59:06,804 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 654
  total_bytes: 28065
}

2021-10-01 09:59:06,905 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 654
  total_bytes: 28065
}

2021-10-01 09:59:07,008 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}

2021-10-01 09:59:07,111 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}

2021-10-01 09:59:07,215 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}

2021-10-01 09:59:07,316 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}

2021-10-01 09:59:07,419 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}

2021-10-01 09:59:07,521 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}

2021-10-01 09:59:07,760 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}

2021-10-01 09:59:08,432 INFO    MainThread:5358 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 28065
  total_bytes: 28065
}
local_info {
}

2021-10-01 09:59:09,587 INFO    MainThread:5358 [wandb_run.py:_show_files():1947] logging synced files
